src/Console/Command/CompletePaidPaymentsCommand.php

```diff
@@ -37,21 +35,9 @@ public function __construct(
         private readonly ObjectManager $paymentManager,
         private readonly CacheAuthorizeClientApiInterface $authorizeClientApi,
         private readonly OrderDetailsApiInterface $orderDetailsApi,
-        private readonly FactoryInterface|StateMachineInterface $stateMachineFactory,
+        private readonly StateMachineInterface $stateMachine,
     ) {
         parent::__construct();
-
-        if ($this->stateMachineFactory instanceof FactoryInterface) {
-            trigger_deprecation(
-                'sylius/paypal-plugin',
-                '1.6',
-                sprintf(
-                    'Passing an instance of "%s" as the fifth argument is deprecated and will be prohibited in 2.0. Use "%s" instead.',
-                    FactoryInterface::class,
-                    StateMachineInterface::class,
-                ),
-            );
-        }
     }
 
     protected function configure(): void
```

Comment:
Replace this configuration method with the attribute

Reply:
I also wonder whether we should not rename this command
```
sylius:paypal:complete-payments
sylius:paypal-plugin:complete-payments
sylius-paypal:complete-payments
```
🤔

Reply:
To be consistent with the `InvoicingPlugin`, I chose `sylius-paypal:complete-payments`.